src/Tizen.TV.UIControls.Forms/Renderer/AccessKeyEffect.cs

```diff
@@ -73,17 +74,19 @@ void OnKeyDown(object sender, EcoreKeyEventArgs e)
 
         bool IsOnCurrentPage(Page currentPage, Page targetPage)
         {
-            if (currentPage == targetPage)
+            var pageToCompare = currentPage.Navigation.ModalStack.Count > 0 ? currentPage.Navigation.ModalStack.LastOrDefault() : currentPage;
```

Comment:
How about split this method?
Check of MadalStack is only need the entry point. 

bool IsOnMainPage(Page targetPage) {
   var pageToCompare = currentPage.Navigation.ModalStack.Count > 0 ? currentPage.Navigation.ModalStack.LastOrDefault() : Application.Current.MainPage;
    return IsOnCurrentPage(pageToCompare, targetPage);
}

bool IsOnCurrentPage(Page currentPage, Page targetPage);